src/hist/hist2root.py

```diff
@@ -0,0 +1,99 @@
+import math
+
+import ROOT
+from coffea import processor
```

Comment:
I would hope hist would be a _dependency_ of coffea, not depending on coffea. This should be done a different way - either optionally importing it, or better yet, if it could be done via a Protocol (check for methods and call if they exist, basically). I'll go through and do a bit of CI / type annotations, then we can revisit this part.